### PR TITLE
snap: add example to update to another channel

### PR DESCRIPTION
**Pull request by SethFalco**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Sadly, tldr didn't tell me how to change an application I already had installed to another channel. :c
Figured since there is space for it, it'd be nice to add.

Referenced the snap docs:
https://snapcraft.io/docs/channels

This command allows a user to change the channel that an already installed package is on. For example, latest/stable → latest/beta or vice versa.

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

**Comment by bl-ue** [comment]
Sorry @SethFalco, typoed `anonther` in the title 😆

File: pages/linux/snap.md [hunk 1]
```diff
@@ -15,6 +15,10 @@
 
 `snap refresh {{package_name}}`
 
+- Update a package to another channel (track, risk-level, or branch):
```
Comment (navarroaxel) [inline pages/linux/snap.md:18]: What is `track, risk-level, or branch`?
```suggestion
- Update a package to another channel:
```

Comment (SethFalco) [reply pages/linux/snap.md:18]: I wasn't sure if the term channel was going to widely understood enough, so I thought that might help make it more explicit.

Track: A type of release, by default it's `latest`, I've never seen this used in any way tbh.
Risk-Level: Stable / Beta / Edge
Branch: For debug releases, like a one-off version of a particular track/risk-level to fix a particular issue or that allows a user to test a specific new feature without giving it to everyone.

I'd like to wait for other opinions, if they agree to remove it, then that's cool.

Comment (navarroaxel) [reply pages/linux/snap.md:18]: Maybe
```suggestion
- Update a package to another channel (e.g. stable, beta or development):
```
Not sure if we should explain the concept of `channel`.

Comment (bl-ue) [reply pages/linux/snap.md:18]: What are the possible valid values?

Comment (navarroaxel) [reply pages/linux/snap.md:18]: It's not fixed, the publisher can use whatever their wants. Node uses `14/stable`, intellij `edge`, `beta`, `stable`.

Comment (bl-ue) [reply pages/linux/snap.md:18]: Ah, I was afraid so.  No suggestions on my part, then :(

Comment (sbrl) [reply pages/linux/snap.md:18]: Generally here we want to use the official terminology that the application itself uses

Comment (SethFalco) [reply pages/linux/snap.md:18]: I assume that means we're deciding between leaving it at channel, or specifying track, risk-level, or branch?

Opinion on each?
1️⃣ Update a package to another channel:
2️⃣ Update a package to another channel (track, risk-level, or branch):
3️⃣ Seth, you misunderstood. Your ideas are bad, and you should feel bad.

Comment (SethFalco) [reply pages/linux/snap.md:18]: Ahh so sad, I was hoping people could react with 1️⃣, 2️⃣, and 3️⃣. Seems we're limited to just the 8 GitHub give us.

Comment (bl-ue) [reply pages/linux/snap.md:18]: Yeah, GitLab is much better in the regard 😢

Comment (sbrl) [reply pages/linux/snap.md:18]: What's the official terminology used by the snapcraft website?

Comment (SethFalco) [reply pages/linux/snap.md:18]: image

In the docs I linked in the PR description

Comment (navarroaxel) [reply pages/linux/snap.md:18]: e.g. `14/stable` for Node.js and `latest/beta`, `2021.1/stable` for IntelliJ. Cool. To make it explicit: The combination of `<track>/<risk>/<branch>` is the name of the channel.

Comment (sbrl) [reply pages/linux/snap.md:18]: In that case, we should probably use that terminology then, to avoid confusing things.

Comment (SethFalco) [reply pages/linux/snap.md:18]: So, we're happy with something like the following?

```suggestion
- Update a package to another channel (track, risk, or branch):
```

**Review by sbrl — APPROVED** [review]
Thanks, @SethFalco!

File: pages/linux/snap.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/snap.md]: Leaving unresolved on purpose, as this looks like an important conversation.

**Pull request merged**